consensus/src/rand/rand_gen/rand_store.rs

```diff
@@ -15,7 +15,7 @@ use aptos_types::randomness::{RandMetadata, Randomness};
 use itertools::Either;
 use std::collections::{BTreeMap, HashMap, HashSet};
 
-const FUTURE_ROUNDS_TO_ACCEPT: u64 = 200;
+const FUTURE_ROUNDS_TO_ACCEPT: u64 = 20;
```

Comment:
Is 20 too aggressive?

Reply:
this is an aggressive chaos test to find something to break, so...

Reply:
But production will also use this config?

Reply:
i don't know. this change and this PR is temporary so far.

Reply:
It will I think